sw/device/tests/BUILD

```diff
@@ -7432,3 +7432,24 @@ opentitan_test(
     ),
     deps = [":rom_exit_immediately_lib"],
 )
+
+opentitan_test(
+    name = "dma_inline_hashing",
+    srcs = ["dma_inline_hashing.c"],
+    exec_env = {
+        "//hw/top_darjeeling:sim_dv": None,
+    },
+    deps = [
```

Comment:
I know it's already transitively included, but please put a `//hw/top_darjeeling/sw/autogen:top_darjeeling` as a reminder that this test is not ported to DT